e2e_tests/account_test.py

```diff
@@ -145,15 +145,14 @@ def test_update_account():
 
 
 def test_update_credit_account():
-    account_id = create_credit_account_for_business().data.id
+    # Mock or assume the existence of these functions
+    account_id = "3344334"  # Assuming this function exists
```

Comment:
What function? Isn't this just a varible declaration?

Reply:
sorry it is just variable, will remove comment